### DISCUSSION TITLE
Connection Error % elif worker.connection_status() == 'offline' %}  Offline

### DISCUSSION THREAD

**Discussion by Sailorderrick** [guydavis/machinaris#680] [General]
Afternoon All

Has anyone seen this error, and how do you go about fixing it?

Connection Error % elif worker.connection_status() == 'offline' %}  Offline

Cheers

**Comment by guydavis** [comment]
Hi, please provide full error report with the context needed to help you out.  For example,

1. A step-by-step break down of how to replicate the error report.  Including a full window screenshot, including browser location bar.
2. A full excerpt of the [log](https://github.com/guydavis/machinaris/wiki/Troubleshooting#where-are-the-logs) in question.
3. Details about your install (Machinaris version, OS platform, etc)

Happy to help with more information.  Thanks.
